### PR TITLE
Minor Type-O

### PR DESCRIPTION
**Pull request by JonTheWong**
Minor correction.

### CODE REVIEW

File: applications/crossbar/priv/couchdb/schemas/user_auth.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
             "type": "string"
         },
         "credentials": {
-            "description": "A hash of the uses credentials",
+            "description": "A hash of the users credentials",
```
Comment (lazedo) [inline applications/crossbar/priv/couchdb/schemas/user_auth.json]:
```suggestion
            "description": "A hash of the user credentials",
```

Comment (JonTheWong) [reply applications/crossbar/priv/couchdb/schemas/user_auth.json]: done

**Comment by JonTheWong** [comment]
Should i update line 35: A phone number assigned to the user account ?

**Comment by lazedo** [comment]
@JonTheWong since this pr was to address this specific typo, there's no need for that. however, you're changing a json schema which in turns requires the docs and the apis to be updated. you should run `make apis` so that you get required changes locally and then you can add them to the pull request.

**Comment by jamesaimonetti** [comment]
I updated the PR with `make apis` corrections and merged. Please run similar (all checks in the build_branch CI task are runnable locally) next time! Thanks for the fix!

**Pull request closed without merging**